setup_wizard/material_data_import_setup/game_material_data_importers.py

```diff
@@ -80,6 +93,40 @@ def find_material_and_outline_material_for_body_part(self, body_part) -> (Materi
 
         return (material, outlines_material)
 
+    def get_material_data_files(self):
+        # Attempt to use the Material or Materials folder in the cached character folder to import material data json
+        # It's possible these folders are in the parent folder for characters with skins, however, it's not possible to
+        # easily determine which material data json to apply to the character, so in that scenario,
+        # pop-up the File Explorer window and ask the user to select material data json files (old flow)
+        cache_enabled = self.context.window_manager.cache_enabled
+        character_directory = self.blender_operator.file_directory \
+            or get_cache(cache_enabled).get(CHARACTER_MODEL_FOLDER_FILE_PATH) \
+            or os.path.dirname(self.blender_operator.filepath)
+        character_material_data_directory = os.path.join(character_directory, 'Material')
+        character_materials_data_directory = os.path.join(character_directory, 'Materials')
+        material_data_directory_exists = os.path.isdir(character_material_data_directory) or \
+            os.path.isdir(character_materials_data_directory)
+        material_data_directory = character_material_data_directory if os.path.isdir(character_material_data_directory) else \
+            character_materials_data_directory if os.path.isdir(character_materials_data_directory) else None
+
+        directory_file_path = os.path.dirname(self.blender_operator.filepath) or material_data_directory
+
+        material_data_files = []
+        if material_data_directory:
+            for filename in os.listdir(material_data_directory):
+                material_data_file = MaterialDataFile(filename)
+                material_data_files.append(material_data_file)
+
+        is_targeted_material_data_import = self.material and self.outlines_material
+        material_data_files = self.blender_operator.files or (material_data_files if not is_targeted_material_data_import else None)
+
+        material_data_file_finder = MaterialDataFileFinder(
+            material_data_directory_exists=material_data_directory_exists,
+            directory_file_path=directory_file_path,
+            material_data_files=material_data_files,
+        )
```

Comment:
Might be better called `MaterialDataDirectory` which would also simplify the field names of the class

```suggestion
        material_data_directory = MaterialDataDirectory(
            exists=material_data_directory_exists,
            file_path=directory_file_path,
            files=material_data_files,
        )
```